.github/workflows/ci.yml

```diff
@@ -42,14 +42,46 @@ jobs:
           install: base-devel mingw-w64-${{matrix.env}}-toolchain
       - run: make
       - run: make test
-  code-coverage:
+  code-coverage-old:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
       - name: Setup LCOV
         uses: hrishikesh-kadam/setup-lcov@v1
       - name: Build and Run tests
         run: make coverage -j
+      # - name: Upload coverage to Codecov
+      #   uses: codecov/codecov-action@v5
+      #   with:
+      #     files: ./cov-html/libopenlibm.info
+      #     token: ${{ secrets.CODECOV_TOKEN }}
+      - uses: actions/upload-artifact@v4
+        with:
+          name: code-coverage-report-old
+          path: ./cov-html/
+  code-coverage:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Openlibm
+        uses: actions/checkout@v4 
+      - name: Checkout Openlibm-test
+        uses: actions/checkout@v4
+        with:
+          repository: 'inkydragon/openlibm-test'
+          path: 'openlibm-test'
```

Comment:
Shouldn't we put these tests in the same org here? I can you give owner on this org if you don't already have it.

Reply:
Also, why not check in the test suite into this repo itself?

Reply:
Let me try it.

Reply:
Moving the repo was successful.
But I seem to have lost write access to https://github.com/JuliaMath/openlibm-test

Reply:
Yes, it works.
I've pushed some commits to main branch.